src/app/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import styled from 'styled-components'
```

Comment:
I cannot see the whole file, do you need styled-components in App.js

Reply:
styled-components in App.js are still a work in progress